bin/hdfcoinc/pycbc_fit_sngls_over_multiparam

```diff
@@ -0,0 +1,129 @@
+parser.add_argument("-V", "--verbose", action="store_true",
+                    help="Print extra debugging information", default=False)
+parser.add_argument("--template-fit-file",
+                    help="Inumpyut hdf5 file containing fit coefficients for each"
+                         " individual template. Required")
+parser.add_argument("--bank-file", default=None,
+                    help="hdf file containing template parameters. Required "
+                         "unless reading param from template fit file")
+parser.add_argument("--output", required=True,
+                    help="Location for output file containing smoothed fit "
+                         "coefficients.  Required")
+parser.add_argument("--use-template-fit-param", action="store_true",
+                    help="Use parameter values stored in the template fit file"
+                         "as template_param for smoothing.", default=False)
+parser.add_argument("--fit-param", nargs='+',
+                    help="Parameter over which to regress the background "
+                         "fit coefficients. Required. Either read from "
+                         "template fit file or choose from mchirp, mtotal, "
+                         "template_duration, a frequency cutoff in pnutils or "
+                         "a frequency function in LALSimulation.")
+parser.add_argument("--approximant", default="SEOBNRv4",
+                    help="Approximant for template duration. Default SEOBNRv4")
+parser.add_argument("--f-lower", type=float, default=0.,
+                    help="Starting frequency for calculating template "
+                         "duration, if not reading from the template fit file")
+parser.add_argument("--min-duration", type=float, default=0.,
+                    help="Fudge factor for templates with tiny or negative "
+                         "values of template_duration: add to duration values"
+                         " before fitting. Units seconds.")
+parser.add_argument("--log-param", nargs='+',
+                    help="Take the log of the fit param before smoothing.")
+parser.add_argument("--smoothing-width", type=float, nargs='+', required=True,
+                    help="Distance in the space of fit param values (or the "
+                         "logs of them) to smooth over. Required. For log "
+                         "template duration, try 0.2")
+args = parser.parse_args()
+pycbc.init_logging(args.verbose)
+
+fits = h5py.File(args.template_fit_file, 'r')
+
+# get the ifo from the template-level fit
+ifo = fits.attrs['ifo']
+
+# get template id and template parameter values
+tid = fits['template_id'][:]
+
+logging.info('Calculating template parameter values')
+bank = h5py.File(args.bank_file, 'r')
+m1, m2, s1z, s2z = trstats.get_masses(bank, tid)
+
+parvals = []
+for param, slog in zip(args.fit_param, args.log_param):
+    data = trstats.get_param(param, args, m1, m2, s1z, s2z)
+    if slog == 'false':
```

Comment:
I suggest to accept both 'false' and 'False' etc., or (better) implement explicit choices in the arg parser